src/ucp/proto/proto_single.inl

```diff
@@ -21,6 +21,7 @@ ucp_proto_am_bcopy_single_send(ucp_request_t *req, ucp_am_id_t am_id,
 {
     ucp_ep_t *ep                       = req->send.ep;
     const uct_iface_attr_t *iface_attr = ucp_ep_get_iface_attr(ep, lane);
+    uct_ep_h uct_ep = fast_path ? ucp_ep_get_fast_lane(ep, lane) : ucp_ep_get_lane(ep, lane);
```

Comment:
how does it fix? looks like code is doing same thing.

Reply:
will fix